src/contribute-to-tidb/report-an-issue.md

```diff
@@ -0,0 +1,48 @@
+
+## Making good issues
+
+Except for a good title and detailed issue message, you can also add suitable labels to your issue via [\label](https://prow.tidb.io/command-help?repo=pingcap%2Ftidb#type), especially which component the issue belongs to. Many committers and contributors only focus on certain subsystems of TiDB. Setting the appropriate component is important for getting their attention.
+
+If you are able to, you should take more considerations on your issue:
+
+  - Does the feature fit well into TiDB's architecture? Will it scale and keep TiDB flexible for the future, or will the feature restrict TiDB in the future?
+  - Is the feature a significant new addition (rather than an improvement to an existing part)? If yes, will the community commit to maintaining this feature?
+  - Does this feature align well with currently ongoing efforts?
+  - Does the feature produce added value for TiDB users or developers? Or does it introduce the risk of regression without adding relevant user or developer benefit? 
+
+Deep thoughts could help the issue proceed faster and help build your own reputation in the community.
+
+## Understanding the issue’s progress and status
+
+Once your issue is created, other contributors might take part in. You can [\cc](https://prow.tidb.io/command-help?repo=pingcap%2Ftidb#cc) reviewers to request reviews. You need to discuss with them, provide more information they might want to know, address their comments to reach consensus and make the progress proceeds. But please realize there are always more pending issues than contributors are able to handle, and especially TiDB community is a global one, contributors reside all over the world and they might already be very busy with their own work and life. Please be patient! If your issue gets stale for some time, it's okay to ping the reviewers and asignees, or take it to [internal.tidb.io](https://internals.tidb.io) for more attention.
+
+## Disagreement with a resolution on the issue tracker
+
+As humans, we will have differences of opinions from time to time. First and foremost, please be respectful that care, thought, and volunteer time went into the resolution.
+With this in mind, take some time to consider any comments made in association with the resolution of the issue. On reflection, the resolution steps may seem more reasonable than you initially thought.
+If you still feel the resolution is incorrect, then raise a thoughtful question on [internal.tidb.io](https://internals.tidb.io). Further argument and disrespectful discourse on [internal.tidb.io](https://internals.tidb.io) after a consensus has been reached amongst the committers is unlikely to win any converts.
+As a reminder, issues closed by a committer have already been carefully considered. Please do not reopen a closed issue.
```

Comment:
```suggestion
As humans, we will have differences of opinions from time to time. First and foremost, please be respectful that care, thought, and volunteer time went into the resolution.

With this in mind, take some time to consider any comments made in association with the resolution of the issue. On reflection, the resolution steps may seem more reasonable than you initially thought.

If you still feel the resolution is incorrect, then raise a thoughtful question on [internal.tidb.io](https://internals.tidb.io). Further argument and disrespectful discourse on [internal.tidb.io](https://internals.tidb.io) after a consensus has been reached amongst the committers is unlikely to win any converts.
```

I think the "reopen" part has already handled by permission that almost only committers can reopen an issue.


BTW, I hope we can have a guide on resolution candidates. cc @bb7133

Reply:
Checked the GitHub permission model, seems issue author seems could only reopen issue closed by themselves but not committers. We can remove the sentence.